cmd/manager/main.go

```diff
@@ -170,8 +170,8 @@ func main() {
 
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
-		Namespace:          namespace,
-		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+		Namespace: namespace,
+		//MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
```

Comment:
to remove

Reply:
I think we will enable some metrics into casskop in near futur ticket I prefer keep this for the moment